compiler/testData/ir/irText/lambdas/destructuringInLambda.txt

```diff
@@ -135,13 +135,13 @@ FILE fqName:<root> fileName:/destructuringInLambda.kt
     FIELD PROPERTY_BACKING_FIELD name:fn type:kotlin.Function1<<root>.A, kotlin.Int> visibility:private [static]
       EXPRESSION_BODY
         FUN_EXPR type=kotlin.Function1<<root>.A, kotlin.Int> origin=LAMBDA
-          FUN LOCAL_FUNCTION_FOR_LAMBDA name:<anonymous> visibility:local modality:FINAL <> (<name for destructuring parameter 0>:<root>.A) returnType:kotlin.Int
-            VALUE_PARAMETER name:<name for destructuring parameter 0> index:0 type:<root>.A
+          FUN LOCAL_FUNCTION_FOR_LAMBDA name:<anonymous> visibility:local modality:FINAL <> ($dstr$_u24__u24$y:<root>.A) returnType:kotlin.Int
```

Comment:
Are these tests only enabled for the JVM_IR backend? Otherwise, I guess this will fail for js and native since the mangling happens in a JVM-specific generator extension.

Reply:
AFAICT yes. There are IR text tests for JS_IR, but the test data for that is under `testData/ir/irJsText` (base class is `AbstractIrJsTextTestCase`). Also AFAICT, no specific tests for Native.